common/djangoapps/student/views/dashboard.py

```diff
@@ -861,6 +868,10 @@ def student_dashboard(request):
         'empty_dashboard_message': empty_dashboard_message,
         'recovery_email_message': recovery_email_message,
         'recovery_email_activation_message': recovery_email_activation_message,
+        # TODO START: Clean up REVEM-205 & REVEM-204.
+        # The below context is for experiments in dashboard_metadata
+        'course_prices': get_experiment_dashboard_metadata_context(course_enrollments) if dashboard_metadata_flag.is_enabled() else None,
+        # TODO END: Clean up REVEM-205 & REVEM-204.
```

Comment:
nitpicking comment: elsewhere we separate this from the building of the context dictionary, and call context.update() to update the context dictionary. See edx-platform/lms/djangoapps/discussion/views.py line 495